### ISSUE TITLE
Server gets stuck while preparing spawn area

### ISSUE THREAD

**Issue by five-red** [devs-immortal/The-Aether#217]
I have been trying to create a server running this mod for my friends for a few hours now however it will not work. I have reinstalled and attempted to recreate the files in various ways (such as launching the server for the first time with the mods and without the mods) to resolve this issue but nothing seems to work. I have left it open for long periods of time but it will not quit via a command, the CMD prompt must be closed manually or it will just sit. 

I am using 

aether-1.16.5-1.0.3
fabric-api-0.34.2+1.16
trinkets-2.6.7

in the mods folder with server.jar version 1.16.5 installed by fabric-installer-0.7.3
[latest.log](https://github.com/devs-immortal/The-Aether/files/6492042/latest.log)

I've attached the log

**Comment by kalucky0** [comment]
Can you try downgrading Fabric API?

**Comment by five-red** [comment]
> Can you try downgrading Fabric API?

I tried this several times, beginning at the latest and working down chronologically in release date and eventually fabric-api-0.33.1+1.16 worked. It launched the server, but the Aether mod appeared to be missing. However, the stop command worked so I shut down the server and relaunched it and it worked fine after that. It did however crash after that first shutdown

[crash-2021-05-17_12.13.30-server.txt](https://github.com/devs-immortal/The-Aether/files/6496316/crash-2021-05-17_12.13.30-server.txt)

However after a few launches I haven't encountered any further issues. 

Attached is the log from the last time I ran the server, where it worked fine if it is of any use. 
[latest.log](https://github.com/devs-immortal/The-Aether/files/6496386/latest.log)
